RetailCoder.VBE/UI/Command/FindAllReferencesCommand.cs

```diff
@@ -100,7 +102,7 @@ protected override bool EvaluateCanExecute(object parameter)
 
         protected override void OnExecute(object parameter)
         {
-            if (_state.Status != ParserState.Ready)
+            if ((_state?.Status ?? ParserState.None) != ParserState.Ready)
```

Comment:
I'm missing the *why* here - have you encountered a case where `_state` was `null`? I'm curious...